lib/pub_sub/configuration.rb

```diff
@@ -5,23 +5,26 @@ class Configuration
                   :subscriptions,
                   :topics,
                   :visibility_timeout,
-                  :idle_timeout,
                   :logger,
                   :regions
 
+    SUPPORTED_REGIONS = %w(us-east-1 us-west-1 eu-west-1 ap-southeast-1)
+
     def initialize
       @subscriptions = {}
       @topics = {}
       # How long to wait before retrying a failed message
       @visibility_timeout = 3600 # seconds, 1 hour
-      # How long to wait before listening on another region
-      @idle_timeout = 60 # seconds
     end
 
     def service(service_name)
       @service_name = service_name.to_s
     end
 
+    def current_region
```

Comment:
There is a wealth of that where regions are received from the client and saved in pubsub cfg. At this point, multiple regions are simply not supported. And I cannot even reference what we used to have as I dropped all that code.